app/static/js/events/wizard/sessions-speakers/vue-components.js

```diff
@@ -7,6 +7,9 @@ Vue.component('session-track', {
         },
         add: function () {
             this.$emit('add');
+        },
+        changefont: function () {
```

Comment:
child component has to emit the event for parent to receive right? Ok I will go for *watch implementation*.